src/app/recipe-details/recipe-details.component.ts

```diff
@@ -20,6 +22,10 @@ export class RecipeDetailsComponent implements OnInit {
 
   recipe: Recipe;
   user: User;
+  ingredients: Recipe[]
```

Comment:
Update this to `ingredients: Array<string>;` , we aren't actually using the model here.